senza/components/elastic_load_balancer.py

```diff
@@ -116,7 +116,9 @@ def component_elastic_load_balancer(definition, configuration, args, info, force
                                               health_check_port,
                                               health_check_path)
 
-    if configuration.get('NameSuffix'):
+    if configuration.get('LoadBalancerName'):
+        loadbalancer_name = info["LoadBalancerName"]
```

Comment:
Of course, it would still be overwritten by the code block at the bottom. You would need to prevent this from happening somehow.